res/values-pt-rBR/strings.xml

```diff
@@ -1309,7 +1309,7 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="skin_warning" translatable="true" tools:ignore="Untranslatable">
         "A sua skin não foi aprovada. Aqui estão algumas das razões mais comuns. Você NÃO deve enviar:
 
-		1. Uma foto de uma pessoa como skin da comunidade. Ela só poderá ser aceita como skin que não seja da comunidade.
+		1. Uma foto de uma pessoa não pode ser usada como skin da comunidade; ela só será aceita se não pertencer à comunidade.
```

Comment:
I advice you to check the exactly same line at my own PR #767.

According to the new guidelines, this rule isn't officially appliable any longer since last year (2023), therefore, it won't make difference if you change it.